src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/ApplicationIniGenerator.scala

```diff
@@ -21,17 +21,25 @@ trait ApplicationIniGenerator {
           val pathMapping = cleanApplicationIniPath(location)
           //Do not use writeLines here because of issue #637
           IO.write(configFile, ("# options from build" +: javaOptions).mkString("\n"))
-          val hasConflict = universalMappings.exists {
-            case (file, path) => file != configFile && path == pathMapping
-          }
-          // Warn the user if he tries to specify options
-          if (hasConflict) {
-            log.warn("--------!!! JVM Options are defined twice !!!-----------")
-            log.warn(
-              "application.ini is already present in output package. Will be overridden by 'javaOptions in Universal'"
-            )
+          val filteredMappings = universalMappings.filter {
+            case (`configFile`, `pathMapping`) =>

```

Comment:
Interesting implementation. I didn't get at first what was happening here. The second call to the method
will filter the already created config file.